PREINSTALL.md

```diff
@@ -0,0 +1,42 @@
+Use this extension to synchronize documents from a Cloud Firestore collection to a Meilisearch index. This allows you to use a full text search in your Cloud Firestore documents.
+
+This extension listens to each creation, update or deletion of your documents to keep them sync with your Meilisearch index, so the data in Meilisearch is a mirror of your content in Cloud Firestore. You can then run queries on this mirrored dataset.
+
+Note that this extension only listens for changes to _documents_ in a specific collection, but not changes in any _subcollection_. However, you can install additional instances of this extension to listen to other collections in your Firestore database.
```

Comment:
Does this mean that if a user has a subcollection, they'll need another extension to export this subcollection to Meilisearch?

Reply:
Yes, you have to create as many collections as you want to mirror index.